I Get Error " Java returned: 1" ?

I Get Error " Java returned: 1" ?

Im clone source 5day  ago...
with last Commits on Dec 4, 2018

> `[java] Dec 09, 2018 4:26:00 AM com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
>      [java] INFO: extracting C:\Users\kangris\AppData\Local\Temp\1544304360407_0.9014596246300873-0\youngandroidproject\project.properties from input zip
>      [java] Component assets needed, n = 2
>      [java] Component activities needed, n = 3
>      [java] Libraries needed, n = 44
>      [java] Native Libraries needed, n = 3
>      [java] Dec 09, 2018 4:26:01 AM com.google.appinventor.buildserver.Compiler generatePermissions
>      [java] INFO: usesLocation = False
>      [java] Permissions needed, n = 96
>      [java] Dec 09, 2018 4:26:01 AM com.google.appinventor.buildserver.Compiler writeAndroidManifest
>      [java] INFO: VCode: 12624800
>      [java] Dec 09, 2018 4:26:01 AM com.google.appinventor.buildserver.Compiler writeAndroidManifest
>      [java] INFO: VName: 2.48
>      [java] java.lang.NullPointerException
>      [java]     at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:191)
>      [java]     at com.google.common.io.Resources$UrlByteSource.<init>(Resources.java:77)
>      [java]     at com.google.common.io.Resources$UrlByteSource.<init>(Resources.java:72)
>      [java]     at com.google.common.io.Resources.asByteSource(Resources.java:66)
>      [java]     at com.google.common.io.Resources.newInputStreamSupplier(Resources.java:57)
>      [java]     at com.google.appinventor.buildserver.Compiler.getResource(Compiler.java:1899)
>      [java]     at com.google.appinventor.buildserver.Compiler.insertNativeLibs(Compiler.java:1725)
>      [java]     at com.google.appinventor.buildserver.Compiler.compile(Compiler.java:995)
>      [java]     at com.google.appinventor.buildserver.ProjectBuilder.build(ProjectBuilder.java:160)
>      [java]     at com.google.appinventor.buildserver.Main.main(Main.java:82)
> 
> BUILD FAILED
> D:\AI\AI2018\appinventor-sources\appinventor\build.xml:21: The following error occurred while executing this line:
> D:\AI\AI2018\appinventor-sources\appinventor\buildserver\build.xml:237: Java returned: 1
> 
> Total time: 14 minutes 51 seconds
> `
> 

How to fix sir?

Thanks

--- a/appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java
+++ b/appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java
@@ -78,6 +78,7 @@ public final class Compiler {
 
   private static final String SLASH = File.separator;
   private static final String COLON = File.pathSeparator;
+  private static final String ZIPSLASH = "/";
 
   public static final String RUNTIME_FILES_DIR = "/" + "files" + "/";
 
@@ -1719,7 +1720,7 @@ public final class Compiler {
           }
 
           String sourcePath = "";
-          String pathSuffix = RUNTIME_FILES_DIR + sourceDirName + SLASH + lib;
+          String pathSuffix = RUNTIME_FILES_DIR + sourceDirName + ZIPSLASH + lib;
 
           if (simpleCompTypes.contains(type)) {
             sourcePath = getResource(pathSuffix);

